src/Hook.php

```diff
@@ -22,10 +22,19 @@ public static function getValidHooks()
         );
         $validHooks = [];
 
+        //add customize hooks
+        $customizeHooks = isset($json['extra']['customize_hooks']) ?
```

Comment:
I want to define some hooks except `applypatch-msg`,`commit-msg` and so on. eg: ctags hook order to read source code.